src/pages/docs/change-workflow.hbs

```diff
@@ -0,0 +1,107 @@
+---
+title: Change Workflow
+---
+<div class=" drizzle-o-Layout__header">
+  <h1 class="drizzle-b-h1">Change Workflow</h1>
+</div>
+
+<div class="drizzle-o-Layout__content drizzle-b-LongFormText">
+  <div class="drizzle-o-ContentGrouping">
+    <p class="drizzle-b-Lede">This is the process the Spark Design System uses to research, design, build, and document Spark components.</p>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+
+    <div class="drizzle-o-ContentGrouping">
+      <h2 class="drizzle-b-h2">Change categories</h2>
+      <p>Every change to Spark will have tasks that fall into at least one of these categrories. Often a change will have tasks in many or all of them. The 4 categories are: <b>Research</b>, <b>Design</b>, <b>Build</b>, and <b>Document</b>.</p>
```

Comment:
Some of these copy areas exceed the max line length. If you wouldn't mind, could you add some returns in?